internal/services/containers/client/client.go

```diff
@@ -37,6 +40,12 @@ type Client struct {
 }
 
 func NewContainersClient(o *common.ClientOptions) (*Client, error) {
+	cacheRulesClient, err := cacherules.NewCacheRulesClientWithBaseURI(o.Environment.ResourceManager)
```

Comment:
I don't think this declaration of the cacheRulesClient is necessary for the new SDK clients anymore